docs/pages/install.mdx

```diff
@@ -48,3 +39,44 @@ For example: 7.1.2
 - 2 represents a bug fix/patch release
 
 The download section above always shows the latest released version of Pelican and we recommend you to use the latest feature or major release version so it includes the latest bug fixes and features. If you would like to download a previous version, please refer to our [GitHub release page](https://github.com/PelicanPlatform/pelican/releases).
+
+## Install Pelican Binary
+
+### Linux
+
+* [Install Pelican on Red Hat Enterprise Linux (CentOS, Fedora, Alma, Rocky, or openSUSE)](./install/rhel.mdx)
+* [Install Pelican on Debian or Ubuntu](./install/debian.mdx)
+* [Install Pelican on Alpine Linux](./install/alpine.mdx)
+
+### macOS
+
+* [Install Pelican on macOS](./install/macos.mdx)
+
+## Verify Pelican Is Installed
+
+> If you installed Pelican as a standalone binary, you need to add Pelican to your `PATH` environment variable before proceeding.
+
+1. Run the following command after you installed Pelican RPM
+
+    ```bash
+    which pelican
+    ```
+
+    It should output
+
+    ```console
+    /bin/pelican
```

Comment:
I don't think this will be true if they install the client on Windows.

Reply:
I think this depends on how the client is "installed" on Windows. 
From the old version, it looks like using on Windows required using a Docker container.
So then the question is whether the Docker container being launched is still a Linux environment; if so, then these instructions are fine.

Reply:
For Windows users, I'd like to have someone test the installation of the Windows binary then decide how I should revise the instruction. I've separate verification for Windows from Linux/macOS, and use `where` command for Windows, but I want to make sure that the Windows installer adds the binary to PATH, otherwise I'll update instruction for it.